### DISCUSSION TITLE
Filterform not working properly for me

### DISCUSSION THREAD

**Discussion by meilensteindigital** [liedekef/events-made-easy#102] [General]
Hi there, i have set up a page with events from multiple categories. on top of this lists i have put the filterform ([events_filterform])  but the filter seems not to work properly. i.e. when i select one category from the dropdown field and hit filter, all other categories are shown as well.
what am i doing wrong? any help is appreciated.

best 
Thomas

here is the link of the page
http://avm-fobi.meilenstein-digital.de/veranstaltungen/
 
here are my event list shortcodes
[events_filterform]
<h2>Präsenzworkshops</h2>
[eme_events category=1 template_id=2 template_id_header=1 template_id_footer=3 paging=10 scope=future order=ASC show_ongoing=1]
<h2>WEBINARE "Friday on my mind"</h2>
[eme_events category=2 template_id=2 template_id_header=1 template_id_footer=3 paging=10 scope=future order=ASC show_ongoing=1]
<h2>WEBINARE "Lectures to go"</h2>
[eme_events category=3 template_id=2 template_id_header=1 template_id_footer=3 paging=10 scope=future order=ASC show_ongoing=1]
<h2>Weitere WEBINARE</h2>
[eme_events category=4 template_id=2 template_id_header=1 template_id_footer=3 paging=10 scope=future order=ASC show_ongoing=1]

and here is my filterform setup
#_FILTER_YEARS{2} #_FILTER_CATS{} #_FILTER_LOCS{Orte} #_FIELD{Dozent} #_SUBMIT{Filter}

**Comment by liedekef** [comment]
When filtering on category, the set category in the eme_events shortcode gets overridden by the selection. So in your case, when selecting a category, all the other eme_events shortcodes on that page will show the same events as well.
Currently this is not seen as a bug, as this was always intended to work this way.

**Reply by meilensteindigital** [reply]
ok i see .. i did'nt even realized that the same events are displayed in all categories *facepalm*. 
then i will put each category on a separate page.

Thank you!!
